client/packages/common/src/intl/locales/en/common.json

```diff
@@ -418,7 +418,7 @@
   "messages.no-scanners-found": "No scanners found",
   "messages.not-applicable": "N/A",
   "messages.not-initialised": "[ Not configured ]",
-  "messages.over-allocated": "Due to the pack sizes available a total quantity of {{allocatedQuantity}} has been allocated rather than {{requestedQuantity}}",
+  "messages.over-allocated": "Due to the pack sizes available a total quantity of {{quantity}} has been allocated rather than {{issueQuantity}}",
```

Comment:
offroad: fixes an issue that @nisha-dangol and I have spotted: [this comment](https://github.com/msupply-foundation/open-msupply/issues/2255#issuecomment-1909870108) has detail